### DISCUSSION TITLE
batch size

### DISCUSSION THREAD

**Discussion by omnyaatef22** [securefederatedai/openfederatedlearning#478] [Q&A]
what is the batch size [indicates?
![Capture55](https://user-images.githubusercontent.com/98268106/183307151-923dbc02-a162-4e6b-b4c3-95279de8875e.PNG)
]

**Comment by daviddelriod** [comment]
The batch size is a common concept used in Deep Learning. It can be describred as the  number of samples from the training dataset used in one forward/backward pass. Setting the batch size to 256 means that in each iteration of training, the model will be trained with 256 samples (in this case, 256 images as you are working with the MNIST dataset).

Additional information:
[https://stats.stackexchange.com/questions/153531/what-is-batch-size-in-neural-network](url)
